torchaudio/functional/filtering.py

```diff
@@ -269,10 +302,17 @@ def biquad(
     device = waveform.device
     dtype = waveform.dtype
 
+    b0 = _float2Tensor(b0, dtype, device).view(1)
+    b1 = _float2Tensor(b1, dtype, device).view(1)
+    b2 = _float2Tensor(b2, dtype, device).view(1)
+    a0 = _float2Tensor(a0, dtype, device).view(1)
+    a1 = _float2Tensor(a1, dtype, device).view(1)
+    a2 = _float2Tensor(a2, dtype, device).view(1)
+
     output_waveform = lfilter(
         waveform,
-        torch.tensor([a0, a1, a2], dtype=dtype, device=device),
-        torch.tensor([b0, b1, b2], dtype=dtype, device=device),
+        torch.cat([a0, a1, a2]),
```

Comment:
Here you could also use [torch.as_tensor](https://pytorch.org/docs/master/generated/torch.as_tensor.html#torch.as_tensor) and construct the array right away.
```
>>> torch.as_tensor([4, torch.tensor(2), 3], dtype=torch.float)
tensor([4., 2., 3.])
```

Reply:
Have tried this approach but it didn't pass `gradcheck` so roll back to use concatenate.